lldb/source/Plugins/Process/Dpu/ProcessDpu.cpp

```diff
@@ -593,7 +602,9 @@ Status ProcessDpu::RemoveBreakpoint(lldb::addr_t addr, bool hardware) {
 
 ProcessDpu::eMemoryAddressSpace
 ProcessDpu::ComputeMemoryAddressSpace(lldb::addr_t addr, size_t size) {
-  if (addr >= k_dpu_iram_base &&
+  if (addr >= k_dpu_iram_base + k_dpu_viram_offset && ((addr & (~k_dpu_viram_msb_mask)) + size) <= (k_dpu_iram_base + m_iram_size)) {
```

Comment:
don't hesitate to \n to be more readable.
group your logical test with ( )
negate ?